deployments/kustomize/base/app/alert-cleanup.yaml

```
@@ -0,0 +1,42 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: alert-cleanup
+spec:
+  schedule: "0 0 * * 0"  # This schedule runs the job once a week on Sunday at midnight.
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: OnFailure
+          volumes:
+            - name: logs-volume
+              persistentVolumeClaim:
+                claimName: naad-logs
+          containers:
+          - name: alert-cleanup
+            image: bcgovgdx/naad-app
+            imagePullPolicy: IfNotPresent
+            envFrom:
+            - configMapRef:
+                name: naad1-config
+            - configMapRef:
+                name: naad-shared-config
+            - configMapRef:
+                name: database-config
+            - secretRef:
+                name: naad-secrets
+            - secretRef:
+                name: database-secrets
+            volumeMounts:
+            - name: logs-volume
+              mountPath: /var/log/app/
+              subPath: naad-1
```

```suggestion
              subPath: cron
```

I think we should separate the actual naad-1 application logs from the cron job logs. This will also require another configmap, something like:

```
  - name: cron-config
    literals:
    - NAAD_NAME=NAAD-CRON
    - NAAD_URL=streaming1.naad-adna.pelmorex.com # Not actually necessary since it doesn't connect to the socket but I think the application will complain if we don't include something here
    - NAAD_REPO_URL=capcp1.naad-adna.pelmorex.com # Same as above
    - LOG_PATH=/var/log/app/cron/app.log
```

---

It shouldn't since it's a mounted volume. That subPath will be unmounted but the log files it writes to should stay on the volume.